pkg/controller/rest/vdri/operation_test.go

```diff
@@ -8,13 +8,17 @@ package vdri
 
 import (
 	"bytes"
+	"encoding/base64"
 	"encoding/json"
 	"fmt"
 	"io"
 	"net/http"
 	"net/http/httptest"
 	"testing"
 
+	mockprovider "github.com/hyperledger/aries-framework-go/pkg/internal/mock/provider"
+	mockstore "github.com/hyperledger/aries-framework-go/pkg/mock/storage"
```

Comment:
These should be grouped with other aries-framework-go imports.

Reply:
done